examples/inabox.host.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <title>amp-inabox-host example</title>
+</head>
+<body>
+  <div style="height: 120vh"></div>
+  <iframe
+      src="http://ads.localhost:8000/examples/inabox.amp.max.html"
+      scrolling="no"
+      width="300" height="200">
+  </iframe>
+  <div style="height: 120vh"></div>
+
+  <script>
+    initInaboxHost(window);
+
+    function initInaboxHost(win) {
+      const hostScriptUrl = 'http://localhost:8000/dist/amp-inabox-host.js';
+      preloadScript(win, hostScriptUrl);
```

Comment:
It's not guaranteed that we'll eventually need inabox scripts, since the ad slot can eventually load a non-AMP ad.

runtime script is much larger host.js, it's rather a question of if we want to sacrifice the bandwidth to boost the speed in case AMP content is loaded.